src/qiskit_iqm/fake_backends/chip_sample.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=no-name-in-module,import-error,too-many-instance-attributes,too-many-arguments,line-too-long
```

Comment:
Why do you need all these ignores? I am particularly surprised to see  `no-name-in-module`, `import-error` and `line-too-long`